### ISSUE TITLE
Error installing as local feed -  OCTGN v 3.3.132.0

### ISSUE THREAD

**Issue by mihnea13** [GeckoTH/arkham-horror#128]
Hey guys,

I'm trying to setup up the game as a local feed in OCTGN. It builds fine (at least, no errors show up) but when trying to load the .nupkg file as a local feed in OCTGN (latest version) it says the feed is invalid. 
Building and installing from the command line, using `o8build.exe. -d -i`, gives more info:

```
Running Test TestFileStructure 
 
Running Test VerifyDef
 Running Test VerifyDefPaths
Running Test VerifyScripts
Running Test TestSetXml
Running Test VerifyProxyDef
Running Test VerifyProxyDefPaths

Building packages
Feed Path: C:\Users\Mihnea\Desktop\arkham-horror-master\o8g\Arkham Horror - The Card Game-1.0.0.40.nupkg
2019-04-25 14:59:05,282 [1] FATAL o8build.Program - Unknown error of doom
 System.NullReferenceException: Object reference not set to an instance of an object.
   at o8build.Program.InstallLocalFeed() in C:\projects\octgn\octgnFX\o8build\Program.cs:line 99
   at o8build.Program.Start() in C:\projects\octgn\octgnFX\o8build\Program.cs:line 93
   at o8build.Program.Main(String[] args) in C:\projects\octgn\octgnFX\o8build\Program.cs:line 37
```

Any ideas?

**Comment by KenDdoox** [comment]
This issue has been resolved now. You can use the new file in the latest PR before it's merged.

**Comment by rushl** [comment]
Which commit resolves this issue?

**Comment by KenDdoox** [comment]
#134 was the one where @ChristopherS pointed out this issue. He made a fix for all the sets that had that problem. After that the sets I made also followed along and there hasn't been any problem, evidenced by the .nupkg file in the latest commit.

**Comment by ChristopherS** [comment]
Sorry I am answering just now. I made a Fix in #134, however the issue described here is a seperate one. He doesn't have problems compiling it, he has problems installing it into the Local Feed. That is actually a problem on the OCTGN side, which we cannot influence. As far as I know they are also aware of that problem. I couldn't find an issue about that though. My suggestion would be to open an issue on the OCTGN repo (https://github.com/octgn/OCTGN) or asking on the OCTGN Discord. 

As far as we are concerned, I would consider this issue closed.
